umbraco/Umbraco.CMS.Backoffice
# Feature: Allow dynamic import of icon for easier bundling

## Description

Currently, we import icons based on a relative path. This causes some trouble when bundling the icon-registry module as files might not be located in the same place after bundling. This PR adds the extra option to "manually" import the icon file (the same way we do in manifests) like `path: () => import('./path/to/icon.js').` This way the bundler knows where the files are located and will handle them correctly.

## What to test
* Make sure that icons render correctly when running with Vite
* Make sure that icons render correctly when building for the CMS.
* All icons are still lazy loaded when they appear on the screen.

## Types of changes
